ui/window.go

```diff
@@ -885,6 +842,50 @@ func NewWindow(doRestart chan bool, app *tview.Application, session *discordgo.S
 	return window, nil
 }
 
+func (window *Window) TrySendMessage(message string) {
+	if window.selectedChannel == nil {
+		return
+	}
+
+	if len(message) == 0 {
+		if window.editingMessageID != nil {
+			msgIDCopy := *window.editingMessageID
+			window.askForMessageDeletion(msgIDCopy, true)
+		}
+		return
+	}
+
```

Comment:
Trim Whitespace here and return if the result is of length `0`  in order to avoid unnecessary API calls.